Should not get a panic for supplying invalid options

Our tools should not panic when supplied an invalid command or option. Currently, however, invalid options for `amp` do. For example, the following (mispelled) option causes a panic.

    $ amp service create --helpp

The CLI should simply display "Unrecognized option: helpp" and exit (1).

Should not panic. Displaying correct usage is fine though.

```
$ amp service create --helpp
Error: unknown flag: --helpp
Usage:  amp service create [OPTIONS] IMAGE [CMD] [ARG...]

Create a new service

Options:
      --config string                 Config file (default is $HOME/.amp.yaml)
      --container-label stringSlice   Set container labels for service replicas (default [])
  -e, --env stringSlice               Set environment variables (default [])
  -h, --help                          help for create
  -l, --label stringSlice             Set service labels (default [])
      --mode string                   Service mode (replicated or global) (default "replicated")
      --name string                   Service name
      --network stringSlice           Set service networks attachment (default [])
  -p, --publish stringSlice           Publish a service externally. Format: [published-name|published-port:]internal-service-port[/protocol], ex: '80:3000/tcp' or 'admin:3000'
      --replicas uint                 Number of tasks
      --server string                 Server address
  -v, --verbose                       Verbose output
      --with-registry-auth            Send registry authentication details to Swarm agents

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5d0b]

goroutine 1 [running]:
panic(0x6ce6e0, 0xc4200140f0)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
main.main()
        /Users/tony/go/src/github.com/appcelerator/amp/cmd/amp/main.go:99 +0x4cb
```

--- a/cmd/amp/main.go
+++ b/cmd/amp/main.go
@@ -96,7 +96,7 @@ func main() {
 		cli.Exit(1)
 	}
 	if err := cmd.Execute(); err != nil {
-		if AMP.Verbose() == false {
+		if verbose {
 			fmt.Printf("Error during: amp %s, reason: %v\n", cmd.Name(), err)
 		}
 		cli.Exit(1)

